### ISSUE TITLE
Namespaces missing after restart

### ISSUE THREAD

**Issue by Ddave182** [blazegraph/database#233]
Hello folks!

I am fairly new to BlazeGraph and wanted to use it in my examination of triple store benchmarks. I therefore created three different namespaces and uploaded 3-4GB of data into each of them, which brought my journal file to a size of 16,6GB. 

However after the restart all namespaces are gone, and running SPARQL queries returns empty results. Though my journal file still holds 16,6GB of triple data.

Any chance on how to restore those lost triples, and namespaces?
Many thanks!

FYI: I downloaded version 2.7.1 from GitHub and ran the .jar file to host the blazegraph server.

**Comment by thompsonbry** [comment]
The file will be extended as incremental writes are pushed through to the
disk. But without a commit there is no record of the root of the indices
for those graphs.

I suggest that you verify that the journal you are reopening is the same
journal that you wrote on.  If you use a different properties file by
mistake it would explain the lack of any loaded data or created namespaces.
That is my best guess as to what is happening here.

Bryan

**Comment by Ddave182** [comment]
Thanks Bryan for your comment and effort. Though i started the server already without any specific properties file, which protected me from selecting the wrong one. I did however restart the server from a different location in my terminal (-server -Xmx4g -jar ~/path/to/file/blazegraph.jar). Only when I navigated back to root directory of Blazegraph and started the server (java -server -Xmx4g -jar ./blazegraph.jar)  from there, all my namespaces reappeared.

Thanks and all the best,
Dave
